backend/inspirehep/assign/tasks.py

```diff
@@ -130,7 +130,7 @@ def assign_conference(record, conference_ref, cnum):
 
 
 @shared_task(
-    ignore_result=False,
+    ignore_result=True,
```

Comment:
This is the export_papers_to_cds one. Even though declared as a task, I can only find it here https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/assign/views.py#L226 called synchronously. Is that the intended behavior ?  @drjova any insights on this ?

Reply:
It's a mistake, it should be triggered as a task in the views

Reply:
Thank you! We can have a small extra PR to fix that @DonHaul